cpp/src/gandiva/tests/test_util.h

```diff
@@ -16,6 +16,7 @@
 // under the License.
 
 #include <chrono>
+#include <filesystem>
```

Comment:
It does not seem this include is used here, is it?

Reply:
Good catch. After moving the logic into `test_util.cc`, the header should be moved into `test_util.cc` as well. Fixed.